ci/sql/estimated-current.sql

```diff
@@ -15,3 +15,21 @@ CREATE TABLE IF NOT EXISTS `globals` (
 ) ENGINE = InnoDB;
 
 INSERT INTO `globals` (frozen_merge_deploy) VALUES (FALSE);
+
+CREATE TABLE IF NOT EXISTS `active_namespaces` (
+  `namespace` VARCHAR(100) NOT NULL,
+  `creation_time` TIMESTAMP NOT NULL DEFAULT (UTC_TIMESTAMP),
+  `expiration_time` TIMESTAMP,
+  PRIMARY KEY (`namespace`)
+) ENGINE = InnoDB;
+
+CREATE TABLE IF NOT EXISTS `deployed_services` (
+  `namespace` VARCHAR(100) NOT NULL,
+  `service` VARCHAR(100) NOT NULL,
+  PRIMARY KEY (`namespace`, `service`),
+  FOREIGN KEY (`namespace`) REFERENCES active_namespaces(namespace) ON DELETE CASCADE
+) ENGINE = InnoDB;
+
+INSERT INTO `active_namespaces` (`namespace`) VALUES (`default`);
+INSERT INTO `deployed_services` (`namespace`, `service`) VALUES
```

Comment:
What about auth-driver? Should this be all possible services including benchmark?

Reply:
My goal here was to restrict to the set of required services in a Hail cluster (that's running CI), which seemed to me like `auth` and `batch`. I don't think the australians are running benchmark, and wouldn't want to muck up their config unnecessarily. Services like benchmark and grafana, etc. will be added in for our clusters once CI does its first deploy.

Regarding `auth-driver`, there's not actually a kubernetes `Service` for that deployment.